app/models/registration.rb

```diff
@@ -552,19 +552,23 @@ def serializable_hash(options = nil)
   end
 
   def self.bulk_auto_accept(competition)
-    if competition.waiting_list.present?
-      competition.registrations
-                 .find(competition.waiting_list.entries)
-                 .each(&:attempt_auto_accept)
+    waitlisted_registrations = competition.registrations.find(competition.waiting_list.entries) if
```

Comment:
This variable assignment isn't necessary, but I think it improves readability of the function